### PR TITLE
Implemented reviewers reviewing without a formal request, now on list

### PR DESCRIPTION
**Pull request by neil-power-hs**
Fixed: Dismissed PRs will now always go to slack.

### CODE REVIEW

File: backend/src/main/resources/sampleJson/reviewDismissed.json [hunk 1]
```diff
@@ -0,0 +1,485 @@
+{
```
Comment (benjamin-hergert-hs) [inline backend/src/main/resources/sampleJson/reviewDismissed.json]: What is this used for? Testing?

Comment (neil-power-hs) [reply backend/src/main/resources/sampleJson/reviewDismissed.json]: yes, planning on adding testing shortly

**Comment by neil-power-hs** [comment]
!hermes

File: backend/src/main/kotlin/com/hootsuite/hermes/github/GithubEventHandler.kt [hunk 2]
```diff
@@ -33,32 +33,30 @@ object GithubEventHandler {
...
-                        DatabaseUtils.createOrUpdateReview(Review.approved(reviewer, prUrl))
-                        SlackMessageHandler.onApproved(reviewer, prAuthor, prUrl)
-                    }
-                    ApprovalState.CHANGES_REQUESTED -> {
-                        DatabaseUtils.createOrUpdateReview(Review.changesRequested(reviewer, prUrl))
-                        SlackMessageHandler.onChangesRequested(reviewer, prAuthor, prUrl, reviewEvent.review.body)
-                    }
-                    ApprovalState.COMMENTED -> {
-                        if (reviewer != author) {
+                PullRequestReviewAction.SUBMITTED -> if (reviewer != author) {
```
Comment (simon-tse-hs) [inline backend/src/main/kotlin/com/hootsuite/hermes/github/GithubEventHandler.kt]: is it me or this nesting getting deeper? Can it be extracted to another function?

File: backend/src/main/resources/sampleJson/reviewDismissed.json (hunk 1 above)
Comment (benjamin-hergert-hs) [reply backend/src/main/resources/sampleJson/reviewDismissed.json]: I'd add the test data when you add the tests

**Pull request merged**